traffic_ops/app/lib/UI/DeliveryService.pm

```diff
@@ -78,7 +78,8 @@ sub get_cdn_domain {
 			distinct => 1
 		}
 	)->get_column('value')->single();
-	return $cdn_domain;
+	# Always return a lowercase FQDN.
+	return lc($cdn_domain);
```

Comment:
does it make sense to lowercase the response from these methods too?

https://github.com/elsloo/incubator-trafficcontrol/blob/master/traffic_ops/app/lib/MojoPlugins/DeliveryService.pm#L213

https://github.com/elsloo/incubator-trafficcontrol/blob/master/traffic_ops/app/lib/MojoPlugins/DeliveryService.pm#L230